docs/usecases.rst

```diff
@@ -20,6 +20,16 @@ Use-cases for libCellML
       a. one component encapsulating two children
       #. one component encapsulating a single child which in turn encapsulates a single child
       #. an invalid cyclical encapsulation hierarchy
+      
+   #. manipulation of a model with multi-level component encapsulation hierarchy
+   
+      a. remove a component
```

Comment:
In fact, looking at what has been implemented so far (see #36), it looks like we should make it clear that we also want to test the removal of a component from a model (and not just a component of sorts).